pkg/queue/prometheus_stats_reporter.go

```diff
@@ -114,10 +112,10 @@ func (r *PrometheusStatsReporter) Report(stat *autoscaler.Stat) error {
 		return errors.New("PrometheusStatsReporter is not initialized yet")
 	}
 
-	operationsPerSecondGV.With(r.labels).Set(stat.RequestCount)
-	proxiedOperationsPerSecondGV.With(r.labels).Set(stat.ProxiedRequestCount)
-	averageConcurrentRequestsGV.With(r.labels).Set(stat.AverageConcurrentRequests)
-	averageProxiedConcurrentRequestsGV.With(r.labels).Set(stat.AverageProxiedConcurrentRequests)
+	operationsPerSecondGV.With(r.labels).Set(stat.RequestCount / r.reportingPeriod.Seconds())
+	proxiedOperationsPerSecondGV.With(r.labels).Set(stat.ProxiedRequestCount / r.reportingPeriod.Seconds())
+	averageConcurrentRequestsGV.With(r.labels).Set(stat.AverageConcurrentRequests / r.reportingPeriod.Seconds())
```

Comment:
I don't think we want to divide average concurrency by reporting period. OPS is a rate over time but concurrency is not.

Reply:
Ugh, great point @yanweiguo, thanks for catching that. Mind fixing it?

Reply:
There you go #5100